README.md

```diff
@@ -804,22 +804,26 @@ Scenario has secrets. Here's how you use them.
 state = scenario.State(
     secrets={
         scenario.Secret(
-            {0: {'key': 'public'}},
-            id='foo',
-        ),
-    },
+            current={'key': 'public'},
```

Comment:
The charm can obtain the revision number of a secret though, can it not? 
Through `secret-info-get` IIRC. Where would that information come from now?

Reply:
> The charm can obtain the revision number of a secret though, can it not? Through `secret-info-get` IIRC.

Yes.

> Where would that information come from now?

The `Secret` object has a `_tracked_revision` and `_latest_revision` that it uses internally, and it returns `_latest_revision` for the `secret-info-get` call. I don't think charms should really be using that value (outside of logging), so there's no public way to set it to something specific. If it turns out I'm wrong about that then it would be easy enough to make these public.

Reply:
I could swear that there was a charm at some point using the secret ID to generate file names.
Or was it relation IDs? 
Either way, I agree it's easier to add than to deprecate and remove, so fine with keeping it hidden for now and add it later if necessary.

Reply:
> I could swear that there was a charm at some point using the secret ID to generate file names. Or was it relation IDs? 

Hmm, I can't find any use of `.revision` like that in the collection of charms I have locally. Maybe it was the ID as you said? Using the ID would still work, just not if you were also using the revision. I guess I could see someone writing a secret to `f"{base_name}-{secret.label}-{secret.revision}"` for the workload to use.

But happy to do it later if required, thanks.